src/assert.js

```diff
@@ -0,0 +1,29 @@
+function Assert() {
```

Comment:
Since we are now using ES6, we wont be using functions in order to form objects, instead we will be always using classes. As I mentioned, you need to define a class with static methods, like this:
```
class Assert {
 static isString...
}